gui/items_list_panel.py

```diff
@@ -31,9 +31,10 @@ def __init__(self, master=None, list_sel_cb=None, bid_cb=None):
         self.new_item_panel.grid(row=2, column=0)
 
     def selection_cb(self, event):
-        self.selected_item = event.widget.get(event.widget.curselection())
-        # See AuctionClientGui's bidding_item_sel_cb function
-        self.list_sel_cb(self.selected_item)
+        if self.items_listbox.curselection():
+            self.selected_item = event.widget.get(event.widget.curselection())
```

Comment:
Change to:

self.selected_item = event.widget.get(event.widget.curselection()[0])